### PR TITLE
0.13 example update: `base-image`

### LINKED ISSUE #3741: 0.13 example update: base-image

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Closes #3741 

**Special notes for your reviewer**:

### CODE REVIEW

File: examples/base-image/backend/garden.yml [hunk 1]
```diff
@@ -1,21 +0,0 @@
-kind: Module
```
Comment (Walther) [inline examples/base-image/backend/garden.yml:1]: considering we have deleted the `garden.yml` file contents, should we rename `backend.garden.yml` to it, or keep the current naming?

File: examples/base-image/README.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
-The `backend` module contains the source code and has two `Dockerfile`s, one for development and one for production. Both `Dockerfile`s extend the image from the `base-image` module, and each sets an `APP_ENV` environment variable that the `backend` returns when called.
+The `backend` application contains the source code and has two `Dockerfile`s, one for development and one for production. Both `Dockerfile`s extend the image from the `base-image` build action, and each sets an `APP_ENV` environment variable that the `backend` returns when called.
 
-To achieve this, we set the `base-image` as a build dependency for the `backend` module:
+To achieve this, we set the `base-image` as a build dependency for the `backend` build action:
 
 ```yaml
-# In backend/garden.yml
-kind: Module
+# In backend/backend.garden.yml
```
Comment (Walther) [inline examples/base-image/README.md:16]: considering we have deleted the `garden.yml` file contents, should we rename `backend.garden.yml` to it, or keep the current naming?

**Review by Walther — APPROVED** [review]
looks good, thanks for the example!

File: examples/base-image/README.md (hunk 2 above)
Comment (vvagaytsev) [reply examples/base-image/README.md:16]: I renamed the file backend/garden.yml to backend/backend.garden.yml. But the renaming appears as a deletion of the old file and an addition of the new one.

We can undo the renamings.

Comment (Walther) [reply examples/base-image/README.md:16]: Ah, makes sense! I have no strong feelings one way or the other, I was just unsure what is the recommended best practice for the filenames.

Comment (vvagaytsev) [reply examples/base-image/README.md:16]: Ok, let's merge it as it is. We can do some renamings later. The config files of the in `demo-project`  were renamed in https://github.com/garden-io/garden/commit/a2320fc704b3a9a05b822e865be2620d05e6fa6f

I just followed the same pattern :)

**Pull request merged**